.cargo/config

```diff
@@ -0,0 +1,3 @@
+[build]
+target = "wasm32-unknown-unknown"
```

Comment:
could you check if these are in a readme file somewhere? I remember we had documentation which mentioned these at some point.

Reply:
Good point, I have a PR I need to get back to making `create-near-app` compatible with Windows. In there I had a command inside the build script, essentially:
`# Note: see flags in ./cargo/config`
I think I'll add it there **and** to the readme, good catch

Reply:
> What do you think about adding a UI to this example? I think the UI from the create-near-app blank project will actually work here. It will make the app a lot better. We need to have more full stack content for rust to make it actually viable. This can be done in a follow up PR.

I agree a UI is a good idea. I am really trying to focus on getting the examples standardized right now, but I think we should open an issue for this. It's actually taking longer than I thought as each example is a teeny bit different.

Reply:
darn it, @janedegtiareva looks like the `.cargo/config` is making tests fail somehow

Reply:
pulled out the `.cargo/config` because it will force those flags onto the tests, which we don't want to do, apparently
https://github.com/rust-lang/cargo/issues/6784